lib/chef/provider.rb

```diff
@@ -286,10 +286,27 @@ def self.include_resource_dsl_module(resource)
           resource.class.properties.each do |name, property|
             class_eval(<<-EOM, __FILE__, __LINE__)
               def #{name}(*args, &block)
+                # not making this a formal deprecation yet because I don't want to drive tim completely crazy, this
+                # should be upgraded to a real deprecation in 13.1 or so
+                Chef::Log.warn "Deprecated property access: please replace calling #{name} with new_resource.#{name}"
```

Comment:
@tas50 what do you think?  if we throw this in as a real honest deprecation warning then if you run with deprecations-are-errors like you like to you'll fail and have to fix all these...

the other alternative is to just not care about it for now and to make sure to remember later.